.gitignore

```diff
@@ -0,0 +1,5 @@
+# dependencies
+node_modules/
+
+# editors
+.vscode/
```

Comment:
This should go into your global gitignore: https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Local gitignores should only ignore project artifacts